### PR TITLE
Addressed Polaris Azure API changes

### PR DESCRIPTION
**Pull request by johan3141592**
- Addressed additional Azure API changes
- Changed the way API changes are handled to reduce the amount of code duplication
- Removed test that modified environment variables, could cause a race condition

### CODE REVIEW

File: pkg/polaris/azure.go [hunk 1]
```diff
@@ -321,7 +321,7 @@ func (c *Client) AzureSubscriptionSetRegions(ctx context.Context, id AzureSubscr
 		Name: subscription.Name,
 	}}
 
-	status, err := c.gql.AzureCloudAccountUpdate(ctx, graphql.CloudNativeProtection, regionAdd, regionRemove, sub)
+	status, err := c.gql.AzureUpdateCloudAccount(ctx, graphql.CloudNativeProtection, regionAdd, regionRemove, sub)
 	if len(status) != 1 {
```
Comment (wdrbk) [inline pkg/polaris/azure.go:325]: Should `err != nil` be checked ahead of the status length?

File: pkg/polaris/azure.go [hunk 2]
```diff
@@ -344,7 +344,7 @@ func (c *Client) AzureSubscriptionSetName(ctx context.Context, id AzureSubscript
 		Name: name,
 	}}
 
-	status, err := c.gql.AzureCloudAccountUpdate(ctx, graphql.CloudNativeProtection, nil, nil, sub)
+	status, err := c.gql.AzureUpdateCloudAccount(ctx, graphql.CloudNativeProtection, nil, nil, sub)
 	if len(status) != 1 {
```
Comment (wdrbk) [inline pkg/polaris/azure.go:348]: Should `err != nil` be checked ahead of the status length?

File: pkg/polaris/azure.go (hunk 1 above)
Comment (johan3141592) [reply pkg/polaris/azure.go]: Agree, should be fixed now.

File: pkg/polaris/azure.go (hunk 2 above)
Comment (johan3141592) [reply pkg/polaris/azure.go]: Agree, should be fixed now.

**Review by wdrbk — APPROVED** [review]

**Pull request merged**